cartographer/BUILD.bazel

```diff
@@ -109,6 +109,16 @@ cc_library(
     ],
 )
 
+cc_binary(
+    name = "cartographer_print_configuration",
+    srcs = ["common/print_configuration_main.cc"],
+    deps = [
+        ":cartographer",
+        "@com_github_gflags_gflags//:gflags",
+        "@com_google_glog//:glog",
+    ],
+)
+
```

Comment:
Bazel builds with this. Don't know if it's good though because I never actually use Bazel.
https://buildkite.com/bazel/cartographer/builds/1537

Reply:
By the way, it looks like other executables (pbstream_main, the groundtruth tools) aren't included in Bazel.